jetty-home/pom.xml

```diff
@@ -145,7 +145,7 @@
                 org.eclipse.jetty.orbit,org.eclipse.jetty.http2,org.eclipse.jetty.websocket,org.eclipse.jetty.fcgi,org.eclipse.jetty.toolchain,org.apache.taglibs
               </excludeGroupIds>
               <excludeArtifactIds>
-                jetty-all,apache-jsp,apache-jstl,jetty-start,jetty-spring
+                jetty-all,jetty-tutorial,apache-jsp,apache-jstl,jetty-start,jetty-spring
```

Comment:
Is there a reason for `jetty-all` to still be there?

Reply:
I can't think of one. 
I'll remove it.